layouts/partials/head.html

```diff
@@ -22,7 +22,7 @@
 {{ with .Site.Params.favicon }}<link rel="icon" href="{{ . | absURL }}" />{{ end }}
 
 <!-- Styles -->
-<link rel="stylesheet" href="{{ "assets/css/split.css" | absURL }}" type="text/css" media="screen" />
+<link rel="stylesheet" href="{{ "/assets/css/split.css" | absURL }}" type="text/css" media="screen" />
```

Comment:
I can't see the difference in the rendered page. It produces the same result. Could you please tell me what the problem is?